files/etc/security/access.conf

```diff
@@ -9,4 +9,4 @@
 # who wants to be able to SSH in as root via public-key on Biomedia servers. 
 # disable SSH for anybody but root
 +:root:ALL
--:ALL EXCEPT (csg) dr jpassera bglocker:ALL
+-:ALL EXCEPT (csg) (biomedia) dr jpassera bglocker jgao:ALL
```

Comment:
regular users shouldn't have SSH access to the cluster nodes, hence the previous config